### PR TITLE
Add extended test for UID 1337

### PR DESCRIPTION
**Pull request by sebrandon1**
Utilizes `tagExtended` to make this run as part of the extended test suite.

Similar to #493 as they are part of the same issue.

Previous PR: #136 

Marked WIP until I can test this.

### CODE REVIEW

**Comment by sebrandon1** [comment]
```

  Begin Captured GinkgoWriter Output >>
    STEP: Testing pods to ensure none are using UID 1337 09/08/22 16:38:54.653
    STEP: checking if pod fail-access has a securityContext RunAsUser 1337 (ns= tnf) 09/08/22 16:38:54.653
    Pod: tnf/fail-access is found to use securityContext RunAsUser 1337
    STEP: checking if pod test-0 has a securityContext RunAsUser 1337 (ns= tnf) 09/08/22 16:38:54.653
    STEP: checking if pod test-1 has a securityContext RunAsUser 1337 (ns= tnf) 09/08/22 16:38:54.653
    STEP: checking if pod test-6d5cd76d4f-747b2 has a securityContext RunAsUser 1337 (ns= tnf) 09/08/22 16:38:54.653
    STEP: checking if pod test-6d5cd76d4f-fqb9g has a securityContext RunAsUser 1337 (ns= tnf) 09/08/22 16:38:54.653
    Number of pods found using UID 1337: 1
  << End Captured GinkgoWriter Output

  Number of pods found using UID 1337: 1
  In [It] at: /Users/bpalm/Repositories/go/src/github.com/test-network-function/cnf-certification-test/cnf-certification-test/accesscontrol/suite.go:727
  ```

Works great, finds the pod I designated with 1337.

File: cnf-certification-test/accesscontrol/suite.go [hunk 1]
```diff
@@ -700,3 +707,23 @@ func testPodRequestsAndLimits(env *provider.TestEnvironment) {
...
 }
+
+func Test1337UIDs(env *provider.TestEnvironment) {
+	// Note this test is only ran as part of the 'extended' test suite.
+	ginkgo.By("Testing pods to ensure none are using UID 1337")
+	const leetNum = 1337
+	var badPods []string
+	for _, put := range env.Pods {
+		ginkgo.By(fmt.Sprintf("checking if pod %s has a securityContext RunAsUser 1337 (ns= %s)", put.Data.Name, put.Data.Namespace))
+		if put.Data.Spec.SecurityContext.RunAsUser != nil && *put.Data.Spec.SecurityContext.RunAsUser == int64(leetNum) {
```
Comment (rdavid) [inline cnf-certification-test/accesscontrol/suite.go:718]: Is it a problem if the pointer is nil?

**Review by rdavid — APPROVED** [review]

**Review by greyerof — APPROVED** [review]

File: cnf-certification-test/accesscontrol/suite.go [hunk 2]
```diff
@@ -700,3 +707,23 @@ func testPodRequestsAndLimits(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+func Test1337UIDs(env *provider.TestEnvironment) {
+	// Note this test is only ran as part of the 'extended' test suite.
+	ginkgo.By("Testing pods to ensure none are using UID 1337")
+	const leetNum = 1337
```
[the remark this answers is not in the document]
Comment (sebrandon1) [reply cnf-certification-test/accesscontrol/suite.go:714]: https://en.wikipedia.org/wiki/Leet

File: cnf-certification-test/accesscontrol/suite.go (hunk 1 above)
Comment (sebrandon1) [reply cnf-certification-test/accesscontrol/suite.go:718]: Yeah it's a safeguard against segfault.

Comment (rdavid) [reply cnf-certification-test/accesscontrol/suite.go:718]: I understand that it prevents null pointer reference. I mean is it en error if the pointer is nil?

File: cnf-certification-test/accesscontrol/suite.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (rdavid) [reply cnf-certification-test/accesscontrol/suite.go:714]: It was worth to put the link to the comment :-)

File: cnf-certification-test/accesscontrol/suite.go (hunk 1 above)
Comment (sebrandon1) [reply cnf-certification-test/accesscontrol/suite.go:718]: Oh, nope in fact it's probably better if its nil overall.

**Pull request merged**